app/static/js/components/Experiments/Experiments.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import NavBar from "components/core/NavBar/NavBar"
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import IngredientFrequency from './IngredientFrequency/IngredientFrequencyBarChart'
+import WordCloud from './WordCloud/WordCloud'
+
+class Experiments extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        experiments: 'ingredient_frequency',
+      };
+    }
+
+    renderExperiment() {
+      const { experiments } = this.state
+      if (experiments === 'word_cloud') {
```

Comment:
```suggestion
      if (experiments === 'word_cloud') {
```
move experiment names into an enum to ensure higher consistency.